packages/itwinui-react/src/styles.js/styles.module.css

```diff
@@ -2,5 +2,8 @@
  * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
  * See LICENSE.md in the project root for license terms and full copyright notice.
  *--------------------------------------------------------------------------------------------*/
-@import '@itwin/itwinui-variables';
-@import '@itwin/itwinui-css';
+
+@layer itwinui-v1, itwinui.v1, itwinui.v2, itwinui.v3;
+
+@import '@itwin/itwinui-variables' layer(itwinui.v3);
+@import '@itwin/itwinui-css' layer(itwinui.v3);
```

Comment:
What does this syntaxes mean/do? Like specifying that the import is part of itwinui.v3 layer?

Reply:
yes, it is importing the stylesheet into the layer: https://developer.mozilla.org/en-US/docs/Web/CSS/@import#importing_css_rules_into_a_cascade_layer

additionally, vite is inlining the contents of the stylesheet using [`postcss-import`](https://github.com/postcss/postcss-import#postcss-import), so the end result looks kinda like this:

```css
@layer itwinui.v3 {
  // ... everything inside itwinui-variables
}
@layer itwinui.v3
  // ... everything inside itwinui-css
}
```

to verify this, you can inspect `styles.css` that gets created after build.